business-central-parent/business-central-distribution-wars/src/main/productized/assembly-kie-wb-swarm-redhat-RHDM.xml

```diff
@@ -20,7 +20,7 @@
           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
 
   <!-- including a . in the id will modify the *classifier* of the artifact, instead of the name/id of the artifact -->
-  <id>swarm-redhat-RHDM</id>
+  <id>thorntail-redhat-RHDM</id>
```

Comment:
could we renamed it to standalone instead of by name of the container as we use for fat jars?